Docs/Book_jp/The_RDKit_Book_jp.rst

```diff
@@ -859,6 +859,8 @@ RDKitを書いている間、コードがマルチスレッド環境でも作動
    コードが``RDK_BUILD_THREADSAFE_SSS`` 引数を使って作られている場合（我々が提供しているバイナリのデフォルトです）、
    一度にスレッド一つだけが与えられた再起的クエリを使っていることを確実に保証するためミューテックスが使われます。
 
+-  複数のスレッドから同時に同じ分子でMolToSmilesを呼び出すと、未定義の動作が発生する可能性があります。
```

Comment:
This is very likely nonsensical. I know very little Japanese. This is google translates best guess.